packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2212,6 +2273,27 @@ class Flexible extends ParentDataWidget<Flex> {
   }
 }
 
+/// A widget that expands a child of a [Row], [Column], or [Flex].
+///
+/// Using an [Expanded] widget to make a child of a [Row], [Column], or [Flex]
+/// expand to fill the available space in the main axis (e.g., horizontally for
+/// a [Row] or vertically for a [Column]). If multiple children are expanded,
+/// the available space is divided amoung them according to the [flex] factor.
+///
+/// An [Expanded] widget must be a descendant of a [Row], [Column], or [Flex],
+/// and the path from the [Flexible] widget to its enclosing [Row], [Column], or
+/// [Flex] must contain only [StatelessWidget]s or [StatefulWidget]s (not other
+/// kinds of widgets, like [RenderObjectWidget]s).
+class Expanded extends Flexible {
+  /// Creates a widget that expands a child of a [Row], [Column], or [Flex]
+  /// expand to fill the available space in the main axis.
+  Expanded({
+    Key key,
+    int flex: 1,
+    @required Widget child,
+  }) : super(key: key, flex: flex, fit: FlexFit.tight, child: child);
```

Comment:
I'd like to do that in a smaller, more targeted patch in cause it causes trouble.